service/history/workflow/update/registry_test.go

```diff
@@ -296,8 +285,8 @@ func TestFindOrCreate(t *testing.T) {
 		})
 
 		t.Run("sending 1st update still denies new update to be created", func(t *testing.T) {
-			_ = upd1.Send(
-				context.Background(), false, &protocolpb.Message_EventId{EventId: testSequencingEventID})
+			t.Helper()
```

Comment:
Is it really needed here?

Reply:
oh, it's not. I inlined a function here and didn't remove this one.